### PR TITLE
Update v1.4 docs

### PR DESCRIPTION
**Pull request by sharmainec**
- Update screenshots
- Update add command

### CODE REVIEW

**Comment by jhlee1997** [comment]
Added Slideshow Class Diagram

File: docs/diagrams/SlideshowClassDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@startuml
```
Comment (sharmainec) [inline docs/diagrams/SlideshowClassDiagram.puml:1]: Should combine with SlideshowAppClassDiagram

File: docs/diagrams/SlideshowAppClassDiagram.puml [hunk 2]
```diff
@@ -17,13 +17,15 @@ Class Slideshow
 Package Flashcard <<Rectangle>>{
 Class Flashcard
 Class Phrase
+Class LanguageType
 }
 
 }
 
 SlideshowApp .up.|> ReadOnlySlideshowApp
 SlideshowApp *--> "1" Slideshow
```
Comment (sharmainec) [inline docs/diagrams/SlideshowAppClassDiagram.puml:26]: Need to include isActive and isAnswerDisplayed

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -368,50 +396,74 @@ The sequence diagram below illustrates the execution of `ListCommand`.
...
-* `AnswerCommand` - When the user enters an answer for the flashcard shown on the current slide.
-* `NextCommand` - When the user navigates to the next slide.
-* `PreviousCommand` - When the user navigates to the previous slide.
-* `StopCommand` - When the user exits slideshow mode.
 
-The `SlideshowApp` class is used to encapsulate all state and operations related to the slideshow.
-It is exposed as a `ReadOnlySlideshowApp` object.
+| Command Class | Command | Usage |
+| `SlideshowCommand` | [`slideshow`](UserGuide/#testing-with-a-set-of-flashcards--slideshow) | When the user enters [Slideshow mode](UserGuide/#slideshow-mode). |
+| `AnswerCommand` | [`answer`](UserGuide/#answering-a-flashcard--answer) | When the user enters an answer for the flashcard shown on the current slide. |
```
Comment (sharmainec) [inline docs/DeveloperGuide.md:413]: Should use flashcard instead of slide

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -479,201 +531,262 @@ University students
...
    Use case ends.
 
 **Extensions**
-* 1a. The given user input is invalid (e.g. `list abc` or `list -1`).
-    * 1a1. LingoGO! shows an error message.
+* 1a. LingoGO! detects that the number of flashcards specified is invalid.
+    * 1a1. LingoGO! informs user that their request is invalid.
 
-      Use case resumes at step 1.
+      Use case resumes from step 1.
```
Comment (sharmainec) [inline docs/DeveloperGuide.md:560]: Use case ends instead of resumes from step 1

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -479,201 +531,262 @@ University students
...
+### Use case: UC06 - Find flashcard(s)
 
-* 4b. The user provides information that causes the flashcard to become a duplicate of another flashcard in LingoGO!
-    * 4b1. LingoGO! shows an error message.
+**MSS**
+1. User requests to find flashcard(s) by providing keyword(s).
+1. LingoGO! shows the flashcard(s) that contain the keyword(s) provided by the user.
 
-      Use case resumes at step 2.
+   Use case ends.
```
Comment (sharmainec) [inline docs/DeveloperGuide.md:635]: Need to indent

File: docs/UserGuide.md [hunk 6]
```diff
@@ -171,6 +203,8 @@ Format: `add l/LANGUAGE e/ENGLISH_PHRASE f/FOREIGN_PHRASE`
 * `ENGLISH_PHRASE` and `FOREIGN_PHRASE` should not be longer than **100 characters**.
   For optimal viewing experience, flashcard phrases should be kept as short as possible.
   If your phrases are too long to be displayed, you can **increase the app's window size**.
+* `LANGUAGE` should not be longer than **50 characters**.
```
Comment (sharmainec) [inline docs/UserGuide.md:206]: Have a `ENGLISH_PRHASE`, `FOREIGN_PHRASE` `LANGUAGE` under Command Words

**Review by sharmainec — COMMENTED** [review]
Remember to change "Use case ends" instead of resumes from step 1 for some of the use cases

**Review by aikenwx — APPROVED** [review]
Fix all outstanding issues in a new branch

**Pull request merged**